test/unit-test/main.tf

```diff
@@ -152,6 +152,87 @@ data "aws_iam_policy_document" "instance-scheduler-lambda-function-policy" {
 resource "aws_lambda_invocation" "test_invocation" {
   function_name = module.module_test.lambda_function_name
 
+  input = jsonencode(
+    {
+      action = "Test"
+  })
+}
+
+# lambda module test with zip package type and vpc config
+module "lambda_function_in_vpc" {
+  source           = "../../"
+  application_name = local.application_name
+  description      = "lambda function provisioned within a vpc test"
+  package_type     = "Zip"
+  filename         = data.archive_file.lambda-zip.output_path
+  source_code_hash = data.archive_file.lambda-zip.output_base64sha256
+  handler          = "test.lambda_handler"
+  runtime          = "python3.8"
+  tags             = local.tags
+  function_name    = "lambda-function-in-vpc-test"
+  create_role      = false
+  lambda_role      = aws_iam_role.lambda-vpc-role.arn
```

Comment:
Could we try to get the order of these arguments similar to the order they appear in the above module block? It helps with readability if properties can be found in a similar place in the list. For example, putting `tags` below `application_name`.

Reply:
resolved